tests/unit/Actions.spec.ts

```diff
@@ -36,8 +36,8 @@ const nrData = {
   },
   consentFlag: 'R',
   corpNum: null,
+  entity_type_cd: 'BC',
```

Comment:
The sample NRs I've seen have both `entity_type_cd` and `requestTypeCd` set to the same value.

Does anyone know for sure which one to look at???